### ISSUE TITLE
Document Ruby version managers

### ISSUE THREAD

**Issue by jdee** [fastlane/docs#443]
I'm opening this because I can't quickly provide all of it myself. It would be good to have some basic documentation for Ruby version managers: Quick setup, common commands, basic usage with Fastlane, etc. I know RVM pretty well. I'm not aware of another brief guide to installing and using it, so I wrote [this](https://github.com/BranchMetrics/fastlane-plugin-branch/wiki/RVM-Cheat-Sheet). I could contribute something similar to Fastlane, but I can't quickly write up rbenv or chruby. Perhaps others would be interested in contributing those guides.

I'd open a PR for the RVM stuff, but if that's the only thing there, it might sound like Fastlane was recommending or endorsing RVM. It might be best to try to get the others covered as well.

**Comment by janpio** [comment]
Good point, I tried fastlane dev without it and that didn't go so well.
Is there maybe a neutral "Use a ruby version manager" site/gist/repo that one could link to?
There should be...
